### PR TITLE
Clean converter Edx to xAPI code

### PR DESCRIPTION
**Pull request by quitterie-lcs**
## Purpose

`school`, `course`, `module` context extensions in Edx to xAPI base converter
are no longer used in xAPI conversion as the course information is now tracked
in context fields depending on the xAPI profile used as a target.
Furthermore, tests for converters need to respect a naming pattern for better
developer experience.

## Proposal

- [x] Remove `school`, `course` and `module` extensions in `BaseXapiConverter`
- [x] Rename converter tests with project path pattern

### CODE REVIEW

**Review by jmaupetit — COMMENTED** [review]
Waiting for CI jobs to pass, else LGTM!

**Review by wilbrdt — APPROVED** [review]
Alt text

**Pull request merged**
